data/sql/updates/pending_db_world/rev_1630085433143335851.sql

```diff
@@ -0,0 +1,33 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1630085433143335851');
+
+-- Add lootid for Ur'dan
+
+DELETE FROM `creature_template` WHERE (`entry` = 14522);
+INSERT INTO `creature_template` (`entry`, `difficulty_entry_1`, `difficulty_entry_2`, `difficulty_entry_3`, `KillCredit1`, `KillCredit2`, `modelid1`, `modelid2`, `modelid3`, `modelid4`, `name`, `subname`, `IconName`, `gossip_menu_id`, `minlevel`, `maxlevel`, `exp`, `faction`, `npcflag`, `speed_walk`, `speed_run`, `scale`, `rank`, `dmgschool`, `DamageModifier`, `BaseAttackTime`, `RangeAttackTime`, `BaseVariance`, `RangeVariance`, `unit_class`, `unit_flags`, `unit_flags2`, `dynamicflags`, `family`, `trainer_type`, `trainer_spell`, `trainer_class`, `trainer_race`, `type`, `type_flags`, `lootid`, `pickpocketloot`, `skinloot`, `PetSpellDataId`, `VehicleId`, `mingold`, `maxgold`, `AIName`, `MovementType`, `InhabitType`, `HoverHeight`, `HealthModifier`, `ManaModifier`, `ArmorModifier`, `RacialLeader`, `movementId`, `RegenHealth`, `mechanic_immune_mask`, `spell_school_immune_mask`, `flags_extra`, `ScriptName`, `VerifiedBuild`) VALUES
+(14522, 0, 0, 0, 0, 0, 14564, 0, 0, 0, 'Ur\'dan', NULL, NULL, 0, 54, 54, 0, 1434, 129, 1, 1.14286, 1, 0, 0, 1, 2000, 2000, 1, 1, 2, 32768, 2048, 0, 0, 0, 0, 0, 0, 7, 0, 14522, 0, 0, 0, 0, 100, 136, '', 0, 3, 1, 1.3, 1, 1, 0, 0, 1, 0, 0, 0, '', 12340);
```

Comment:
It looks like this code is just trying to set the lootid to a nonzero value. is there a reason this can't be done with a regular `UPDATE`?

Reply:
As for this, i can't use keira3 for changing drop rates, because lootid was 0 value.
So i just add lootid that same as entry, if you have a reference, I'm happy to take a look or  i will try first then

Reply:
Oh that's cool, I'm not familiar with Keira but I just looked it up and it looks like you can use it to generate SQL like this.  That's pretty cool! If you need help cleaning up the SQL a bit, I'd be happy to work with you on it, just look for me in the Azeroth Core Discord and send me a message